csaf_2.1/prose/edit/src/schema-elements-01-defs-03-full-product-name.md

```diff
@@ -325,9 +331,23 @@ Any given serial number of value type `string` with at least 1 character represe
 abbreviated (partial) serial number of the component to identify.
 
 If a part of a serial number of the component to identify is given,
-it SHOULD begin with the first character of the serial number and stop at any point.
-Characters which SHOULD NOT be matched MUST be replaced by either `?` (for a single character) or `*` (for zero or more characters).  
-Two `*` MUST NOT follow each other.
+it MUST begin at the first and end at the last character position of the string representing the targeted component.
+The wildcard characters `?` (for a single character) and `*` (for zero or more characters) signal exclusion of characters at these positions from matching.
```

Comment:
I'm not sure, whether the sentence is quite clear. What we actually want is to tell that for a partial number you have to give the full structure but may replace irrelevant parts with the wildcard characters. What do you think about this?
```suggestion
To provide a part of a serial number of the component to identify, the full structure of the serial number MUST be given.
However, characters that are irrelevant to the matching MAY be replaced by the wildcard characters `?` (for a single character) and `*` (for zero or more characters) that signal exclusion of characters at these positions from matching.